src/Calendar/Calendar.js

```diff
@@ -564,6 +570,11 @@ Calendar.basePropTypes = {
 Calendar.propTypes = {
     ...Calendar.basePropTypes,
 
+    localizedText: CustomPropTypes.i18n({
+        daysOfWeek: PropTypes.arrayOf(PropTypes.string),
+        monthsOfYear: PropTypes.arrayOf(PropTypes.string),
+        singleCharDaysOfWeek: PropTypes.arrayOf(PropTypes.string)
+    }),
```

Comment:
It _seems_ wrong to pass localized text as an array of strings.  I know a calendar is just a table with rows and columns, but we have gotten bitten before by just sending localized text strings in as arrays for API simplicity.  If the code within the component assembles the strings into arrays for construction purposes, that's fine, but I think localized text strings should come in as individual strings.